src/libcommonserver/log/log.h

```diff
@@ -161,11 +161,50 @@ class COMMONSERVER_EXPORT Log {
         inline log4cplus::Logger getLogger() { return _logger; }
         bool configure(bool useLog, LogLevel logLevel, bool purgeOldLogs);
 
+        /*! Returns the estimated size of the log files in bytes.
```

Comment:
Is it the size of the compressed archive, or the size of the uncompressed log files?

Reply:
It is the size of the log directory. Which is approximately the size of the archive.
The log directory contains mainly already compressed files for the old logs (https://github.com/Infomaniak/desktop-kDrive/pull/100#discussion_r1592040458) and two non compress file for the logs of the current session. 
The goal here is not to have a precise size but an idea of the amount of data to upload.

Reply:
I am moving all these functions in a new class (LogArchiver). I renamed the function to getLogDirEstimatedSize in it 👍🏾.